run-tests-with-build-kogito-examples/pom.xml

```diff
@@ -12,47 +12,38 @@
   <artifactId>run-tests-with-build-kogito-examples</artifactId>
 
   <properties>
-    <sources.url>${sources.url}</sources.url>
+    <!-- integration-tests-quarkus-decisions tests depends on src/main/java classes -->
```

Comment:
Could we please update the comment?

`integration-tests-quarkus-decisions` refers to submodule of `kogito-runtimes`.

In the context of `kogito-examples`, this is needed for example due to submodule `dmn-listener-quarkus`, as there are some java classes under `src/main/java` used during test runtime.